.github/workflows/manage-azure-policy.yml

```
@@ -6,7 +6,7 @@ on:
   pull_request: 
 
 env: 
-  MANAGEMENT_GROUP: dts002
+  MANAGEMENT_GROUP: HMCTS Programmes
```

ID is HMCTS
```suggestion
  MANAGEMENT_GROUP: HMCTS
```

what does this management group actually do though?